### PR TITLE
docs: Update permissions section in user guide

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Some users were confused about how to use custom vs device permissions, so this tries to clarify that.

# Checklist
- [x] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

File: js-miniapp-sdk/README.md [hunk 1]
```diff
@@ -71,44 +71,38 @@ miniApp.getUniqueId()
...
-	.then(success => {
-		console.log(success); // Allowed.
-	}).catch(error => {
-		console.error(error); // Permission is not granted due to many circumstances.
-    });
-```
+These permissions are requested using the [MiniAppFeatures.requestCustomPermissions](api/interfaces/miniappfeatures.md#requestcustompermissions) method.
+
+| Permission Name | Description |
+| ---- | ---- | ---- |
```
Comment (khairul-alam-licon) [inline js-miniapp-sdk/README.md:91]: One minor issue: Maybe this part is broken? We can use `| ---- | ---- |`

Comment (corycaywood) [reply js-miniapp-sdk/README.md]: Thanks, I updated this.

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Review by minh-rakuten — APPROVED** [review]
👍 👍

**Pull request merged**